src/avatar-group.scss

```diff
@@ -81,6 +81,19 @@ $block: #{$fd-namespace}-avatar-group;
     }
   }
 
+  &__overflow {
```

Comment:
why do you need this? It's used like this `<div class="fd-popover fd-avatar-group__overflow">` and it already has a reset coming from `fd-popover`. If you inspect the element, all of the rules are overwritten:
<img width="431" alt="Screen Shot 2021-02-05 at 2 23 54 PM" src="https://user-images.githubusercontent.com/39598672/107079410-d4b45500-67bd-11eb-9b70-57e8e6ff1c94.png">

You can only use `fd-avatar-group__overflow-body` where you apply the 1rem padding and use the specificity to add margin to the item.

Reply:
@InnaAtanasova fixed